src/layers/microscopy-viewer-layer/microscopy-viewer-layer-base.js

```diff
@@ -3,15 +3,13 @@ import { COORDINATE_SYSTEM } from '@deck.gl/core';
 import { XRLayer } from '../xr-layer';
 import { tileToScreen, getRasterTileIndices } from './tiling-utils';
 
-import { loadZarr, loadTiff } from './data-utils';
-import {
-  padWithDefault,
-  setOrderedValues,
-  DEFAULT_COLOR_OFF,
-  DEFAULT_SLIDER_OFF
-} from './utils';
+import { padWithDefault } from './utils';
 
 const MAX_SLIDERS_AND_CHANNELS = 6;
+const MAX_SLIDER_VALUE = 65535;
```

Comment:
I think this is as good a time as any to convert this into some sort of prop - it can be inferred from the data type of the loader.

Reply:
ok